### PR TITLE
Added "Award this map on TMX"

### PR DESCRIPTION
**Pull request by nbeerten**
tldr:
- added award this map on TMX button to the button list on a map page
- changed map name from `UI::Text` to `UI::TextWrapped`

On maps tabs, I added a "Award this map on TMX" button. The button leads them to the map page like usual, but with a `#award` hash, which makes a pop-up show up prompting them to award the map. The main benefit of this is the reminder to award a map, as it's a good way to somewhat support a mapper. 

And I noticed the map name was `Text` instead of `TextWrapped`, which made long names clip outside of the window instead of wrap, so I changed that as well (during my attempts at finding a decent spot for the award button).

### CODE REVIEW

File: src/Interface/Tabs/Map.as [hunk 1]
```diff
@@ -405,6 +405,14 @@ class MapTab : Tab
         UI::SetPreviousTooltip("Mood");
         UI::Text(Icons::Money + " \\$f77" + m_map.DisplayCost);
         UI::SetPreviousTooltip("Coppers cost");
+
+        /* Inlined styles, because this is the only place where a gold button is needed */
+        UI::PushStyleColor(UI::Col::Button, UI::HSV(0.12f, 1, 0.7f));
+        UI::PushStyleColor(UI::Col::ButtonHovered, UI::HSV(0.12f, 1.1f, 0.8f));
+        UI::PushStyleColor(UI::Col::ButtonActive, UI::HSV(0.12f, 1.2f, 0.9f));
```
Comment (GreepTheSheep) [inline src/Interface/Tabs/Map.as:412]: Can you use `UI::ColoredButton()` or something like that instead of pushing style colors?
The method is listed in src/Utils/UI

**Review by GreepTheSheep — CHANGES_REQUESTED** [review]
You need changes for the colored button. Thanks in advance

**Comment by nbeerten** [comment]
Fixed now. Added `UI::GoldButton()` to [`src/Utils/UI/ColoredButton.as`](https://github.com/nbeerten/openplanet-maniaexchange-menu/blob/d98500803b798fc1ff1c6ee8139790f2c846f0aa/src/Utils/UI/ColoredButton.as#L20).

**Review by GreepTheSheep — APPROVED** [review]

**Comment by GreepTheSheep** [comment]
Thanks
